app/src/main/res/layout/activity_main.xml

```diff
@@ -1,18 +1,204 @@
+        android:background="@color/main_background"
+        android:orientation="vertical">
 
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+        <TextView
+            android:id="@+id/calculation_label"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@color/label_background"
+            android:hint="@string/label"
+            android:maxLines="@integer/max_lines_label"
+            android:textSize="@dimen/activity_calculation_label_text_size"
+            app:layout_constraintBottom_toTopOf="@+id/input_label"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/input_label"
+            android:layout_width="wrap_content"
+            android:layout_height="@dimen/activity_input_label_height"
+            android:background="@color/label_background"
+            android:hint="@string/label"
+            android:maxLines="@integer/max_lines_label"
+            android:textSize="@dimen/activity_input_label_text_size"
+            app:layout_constraintBottom_toTopOf="@id/button_main_clear"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/calculation_label" />
+
+        <Button
+            android:id="@+id/button_main_clear"
+            style="@style/clearButtonStyle"
+            android:text="@string/clear"
+            app:layout_constraintBottom_toTopOf="@id/button_main_number_seven"
+            app:layout_constraintEnd_toStartOf="@id/button_main_delete_current_number"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/input_label" />
+
+        <Button
+            android:id="@+id/button_main_delete_current_number"
+            style="@style/clearButtonStyle"
+            android:text="@string/delete_button"
+            app:layout_constraintBottom_toTopOf="@id/button_main_number_eight"
+            app:layout_constraintEnd_toStartOf="@id/button_main_divide"
+            app:layout_constraintStart_toEndOf="@id/button_main_clear"
+            app:layout_constraintTop_toBottomOf="@id/input_label" />
+
+        <Button
+            android:id="@+id/button_main_number_one"
+            style="@style/numberButtonStyle"
+            android:text="@string/button_one"
+            app:layout_constraintBottom_toTopOf="@id/button_main_number_zero"
+            app:layout_constraintEnd_toStartOf="@id/button_main_number_two"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/button_main_number_four" />
+
+        <Button
+            android:id="@+id/button_main_number_two"
+            style="@style/numberButtonStyle"
+            android:text="@string/button_two"
```

Comment:
Pls , rename the string resouces with another more representative.

Reply:
done, thanks